pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -53,6 +55,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to secrets being used by hosts
+	err = c.Watch(&source.Kind{Type: &v1.Secret{}},
+		&handler.EnqueueRequestForOwner{
+			IsController: true,
+			OwnerType:    &metalkubev1alpha1.BareMetalHost{},
+		})
+	if err != nil {
```

Comment:
As a nit, I would make this line `return err` and remove the remaining lines in the function.

Reply:
Done